frontend/src/components/Wheel/index.css

```diff
@@ -0,0 +1,30 @@
+.circle {
+  width: 25em;
+  height: 25em;
+  border: 1px solid black;
+  position: relative;
+  padding: 0;
+  margin: 1em auto;
+  border-radius: 50%;
+  list-style: none;
+  overflow: hidden;
+}
+
+.start-rotate {
+  animation: start-rotate 1s linear infinite;
+}
+
+@keyframes start-rotate {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.stop-rotate {
+  animation-play-state: paused;
+}
+
+.img_prizes {
+  width: 30px;
+  height: 30px;
+}
```

Comment:
We should avoid inconsistent naming conventions: `img_prizes` vs `stop-rotate` for example.